DVSRegister/Startup.cs

```diff
@@ -27,8 +27,19 @@ public Startup(IConfiguration configuration, IWebHostEnvironment webHostEnvironm
         {
             this.configuration = configuration;
             this.webHostEnvironment = webHostEnvironment;
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            app.UseMiddleware<SecurityHeadersMiddleware>();
+            app.UseStaticFiles();
```

Comment:
move   app.UseMiddleware<SecurityHeadersMiddleware>();   to program.cs before app.UseHttpsRedirection();